cases/cosmo-ghg-11km-test/config.py

```diff
@@ -141,7 +149,8 @@
 # SIMULATION =================================================================
 # COSMO ----------------------------------------------------------------------
 # Executable
-cosmo_bin = os.path.join(exe_dir, "cosmo-org-ghg-pgi_2b2a54e7_20210127")
+cosmo_bin = os.popen('spack location -i cosmo@empa-ghg%nvhpc').read().strip(
+) + '/bin/cosmo-ghg_gpu'
```

Comment:
Is this command going to work when spack hasn't been initialized with `source /project/g110/spack/user/daint/spack/share/spack/setup-env.sh`? Or is spack somehow initialized when activating the Conda environment?

Reply:
According to the README, spack is initialized after activating the environment and before running the tests. However, there is a more elegant solution with `spack load cosmo@empa-ghg%nvhpc`. Then the path to the executable will be set as environment variable, which means we can write `cosmo_bin = cosmo-ghg_gpu`. In the COSMO runscript, we just have to initialize the spack environment there. Something similar has been done with `icontools`.
I will add this change.

Reply:
OK. I'd suggest you update the README to make initializing spack a more clear requirement for running the chain for COSMO jobs. Right now it seems only required to build COSMO and not otherwise.

Reply:
Normally, we could hide the initialization of spack in the runscript directly. However, this is not possible with int2lm as no executable `int2lm` is created. Thus, we indeed need to have spack initialized when running COSMO jobs. I added a note to the README.

Reply:
Great! 👍